Build/GitHooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+npm run lint
```

Comment:
I'm not sure, if we can afford to do this on a per-commit basis anymore. With lerna this takes up to 10 minutes. Though, maybe this time decreases when playing a litle with lernas concurrency flag...